test/test_publisher.py

```diff
@@ -45,12 +45,15 @@ def setUpClass(self):
         self.config['confluence_publish'] = False
         self.config['confluence_purge'] = not self.single_docname
 
-        self._conf('confluence_parent_page',    'CB_PAR', DEFAULT_TEST_PARENT)
-        self._conf('confluence_publish_prefix', 'CB_PFX')
-        self._conf('confluence_server_pass',    'CB_PWD')
-        self._conf('confluence_server_url',     'CB_URL', DEFAULT_TEST_URL)
-        self._conf('confluence_server_user',    'CB_USR', DEFAULT_TEST_USER)
-        self._conf('confluence_space_name',     'CB_SPC', DEFAULT_TEST_SPACE)
+        self._conf('confluence_parent_page',      'CB_PAR', DEFAULT_TEST_PARENT)
+        self._conf('confluence_publish_prefix',   'CB_PFX')
+        self._conf('confluence_server_pass',      'CB_PWD')
+        self._conf('confluence_server_url',       'CB_URL', DEFAULT_TEST_URL)
+        self._conf('confluence_server_user',      'CB_USR', DEFAULT_TEST_USER)
+        self._conf('confluence_space_name',       'CB_SPC', DEFAULT_TEST_SPACE)
+        self._conf('confluence_ca_cert',          'CB_CA')
```

Comment:
(note) Heads up that the `test_publisher.py` class is about to be re-written. You can leave these changes in if you'd like, but just note these changes may or may not exist in the future (if it matters to you).